projects/1.18-fabric/assets/twigs/twigs/lang/zh_cn.json

```
@@ -0,0 +1,150 @@
+	"block.twigs.polished_bloodstone":"磨制血石",
+	"block.twigs.polished_bloodstone_slab":"磨制血石台阶",
+	"block.twigs.polished_bloodstone_stairs":"磨制血石楼梯",
+	"block.twigs.polished_bloodstone_bricks":"磨制血石砖",
+	"block.twigs.polished_bloodstone_brick_wall":"磨制血石砖墙",
+	"block.twigs.polished_bloodstone_brick_slab":"磨制血石砖台阶",
+	"block.twigs.polished_bloodstone_brick_stairs":"磨制血石砖楼梯",
+	"block.twigs.cracked_polished_bloodstone_bricks":"磨制裂纹血石砖",
+	"block.twigs.rhyolite":"流纹岩",
+	"block.twigs.rhyolite_wall":"流纹岩墙",
+	"block.twigs.rhyolite_slab":"流纹岩台阶",
+	"block.twigs.rhyolite_stairs":"流纹岩楼梯",
+	"block.twigs.polished_rhyolite":"磨制流纹岩",
+	"block.twigs.polished_rhyolite_slab":"磨制流纹岩台阶",
+	"block.twigs.polished_rhyolite_stairs":"磨制流纹岩楼梯",
+	"block.twigs.polished_rhyolite_bricks":"磨制流纹岩砖块",
+	"block.twigs.polished_rhyolite_brick_wall":"磨制流纹岩砖墙",
+	"block.twigs.polished_rhyolite_brick_slab":"磨制流纹岩砖台阶",
+	"block.twigs.polished_rhyolite_brick_stairs":"磨制流纹岩砖楼梯",
+	"block.twigs.cracked_polished_rhyolite_bricks":"磨制裂纹流纹岩砖块",
+	"block.twigs.tuff_wall":"凝灰岩墙",
+	"block.twigs.tuff_slab":"凝灰岩台阶",
+	"block.twigs.tuff_stairs":"凝灰岩楼梯",
+	"block.twigs.cracked_polished_tuff_bricks":"裂纹磨制凝灰岩砖块",
+	"block.twigs.polished_tuff":"磨制凝灰岩",
+	"block.twigs.polished_tuff_slab":"磨制凝灰岩台阶",
+	"block.twigs.polished_tuff_stairs":"磨制凝灰岩楼梯",
+	"block.twigs.polished_tuff_bricks":"磨制凝灰岩砖块",
+	"block.twigs.polished_tuff_brick_wall":"磨制凝灰岩砖墙",
+	"block.twigs.polished_tuff_brick_slab":"磨制凝灰岩砖台阶",
+	"block.twigs.polished_tuff_brick_stairs":"磨制凝灰岩砖楼梯",
+	"block.twigs.calcite_wall":"方解石墙",
+	"block.twigs.calcite_slab":"方解石台阶",
+	"block.twigs.polished_calcite":"磨制方解石",
+	"block.twigs.cracked_polished_calcite_bricks":"裂纹磨制方解石砖块",
+	"block.twigs.polished_calcite_slab":"磨制方解石台阶",
+	"block.twigs.polished_calcite_stairs":"磨制方解石楼梯",
+	"block.twigs.polished_calcite_bricks":"磨制方解石砖块",
+	"block.twigs.polished_calcite_brick_wall":"磨制方解石砖墙",
+	"block.twigs.polished_calcite_brick_slab":"磨制方解石砖台阶",
+	"block.twigs.polished_calcite_brick_stairs":"磨制方解石砖楼梯",
+	"block.twigs.calcite_stairs":"方解石楼梯",
+	"block.twigs.stripped_bamboo":"去皮竹子",
+	"block.twigs.stripped_bamboo_planks":"去皮竹板",
+	"block.twigs.stripped_bamboo_door":"去皮竹门",
+	"block.twigs.stripped_bamboo_trapdoor":"去皮竹活板门",
+	"block.twigs.stripped_bamboo_slab":"去皮竹台阶",
+	"block.twigs.stripped_bamboo_stairs":"去皮竹楼梯",
+	"block.twigs.stripped_bamboo_pressure_plate":"去皮竹压力板",
+	"block.twigs.stripped_bamboo_button":"去皮竹按钮",
+	"block.twigs.stripped_bamboo_fence":"去皮竹栅栏",
+	"block.twigs.stripped_bamboo_fence_gate":"去皮竹栅栏门",
+	"block.twigs.rocky_dirt":"碎石土",
+	"block.twigs.crimson_shroomlamp":"绯红菌光灯",
+	"block.twigs.warped_shroomlamp":"诡异菌光灯",
+	"block.twigs.lamp":"灯",
+	"block.twigs.soul_lamp":"灵魂灯",
+	"block.twigs.paper_lantern":"纸灯笼",
+	"block.twigs.allium_paper_lantern":"饰纹纸灯笼",
+	"block.twigs.dandelion_paper_lantern":"饰纹纸灯笼",
+	"block.twigs.crimson_roots_paper_lantern":"饰纹纸灯笼",
+	"block.twigs.blue_orchid_paper_lantern":"饰纹纸灯笼",
+	"block.twigs.chiseled_bricks":"雕纹红砖块",

```

```suggestion
	"block.twigs.chiseled_bricks":"錾制红砖块",
```
此改动是在1193进行的，1182请用`錾制`
全文通查